docs/Reverse-Paging.md

```diff
@@ -0,0 +1,38 @@
+# Implementation of Reverse Paging
+
+This is the result documentation of [APIPUB-62](https://edfi.atlassian.net/browse/APIPUB-62)  
+
+## In relation to how Publisher does pagination
+
+Publisher does pagination in 2 levels. The first one is done using the Change Version feature from the Web Api. The second one is done using the offset and limit parameters.
+
+At a high level, the execution it works like this.
+	1. Get the total number of records.
```

Comment:
The list doesn't display correctly in the page
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Publisher/blob/APIPUB-68-2/docs/Reverse-Paging.md